src/wlapplication_messages.cc

```diff
@@ -88,8 +88,8 @@ void fill_parameter_vector() {
 		  "description of the AI training logic"),
 		false},
 	  {"", "auto_speed", "",
-		_("In multiplayer games only, this will keep adjusting the game speed "
-		  "automatically, depending on FPS. Useful in conjunction with --ai_training"),
+		_("This will keep adjusting the game speed automatically, depending on AI delay."
+		  "Only to be used for AI testing or training (in conjunction with --ai_training)"),
 		false},
```

Comment:
I think now this option will be used by about 0.001% of players, so the `ai_training` and `auto_speed` options should be made Verbose options by changing `false` to `true` here and in line 89